Arreglos

Clarissa0512 opened the pull request:
En este conjunto de trabajos he arreglado diferentes casos en el proyecto de Paulo Coelho, tanto en el document de css como en el de html. Los problemas arreglados tenían diferentes casos como posiciones de algunos códigos, algunos espacios y códigos innecesarios.

Clarissa0512 committed:
Pegando los selectores al margen.

Aquí he pegado los selectores al margen derecho del editor.

Clarissa0512 committed:
Añadiendo espacios.

Aquí añado espacios entre los elementos y las respectivas llaves, además colocar las llaves cerradas en una línea propia.

Clarissa0512 committed:
Poniendo los selectores grupo en filas...

Aquí coloco los selectores que están en grupo, cada uno en una fila.

Clarissa0512 committed:
Añadiendo espacios despues dos puntos.

Aquí añado espacios después de algunos a los cuales les faltaba.

Clarissa0512 committed:
Quitar los espacios a los colores rgb().

Aquí remuevo los espacios de los colores de tipo rgb().

Clarissa0512 committed:
Remover px de los ceros.

Aquí les quito los px a los ceros.

Clarissa0512 committed:
Indentar los elementos.

Aquí indento los elemetos de css con dos espacios.

Clarissa0512 committed:
Remover el type.

Ahora removí el type de el link.

misscs commented on coelho.html:
```diff
@@ -1,39 +1,39 @@
+	  </div>
+	<footer>
+	  <ul>
+		  <li><a href="file:///C:/Myhtml-C/my-proyecto/yoko-kitchen/autobiografia.html">
```
Aquí hay outro cambio que debes notar para corregir. El dirección del `href` está incorrecta. Has subido ese proyecto a Github?

Clarissa0512 replied:
si.

misscs commented on coelho.html:
```diff
@@ -1,39 +1,39 @@
+	<link rel= "stylesheet" href="coelho.css"/>
+  </head>
+  <body>
+      <div>
```
Parece que este `div` no está cerrado. Es extra?

Clarissa0512 replied:
mmm Si está cerrado.

misscs commented on coelho.html:
```diff
@@ -1,39 +1,39 @@
+  <header>
+		  <h1><i>Estos son escritos</i><em>de</em><b>Paulo Coelho</b></h1>
+  </header>
+		  <hr></hr>
```
El `<hr`> es como `<img>` :)

Clarissa0512 replied:
Es cierto...

misscs commented:
MUY BIEN!!! Al revisar, encontré algunas problemas chiquitas con la direction del `href` y `hr`. 

Este PR es uno de los mejores que has hecho! Puedo seguir la historia del PR sin problema. Y como cada commit solamente mejoraba una cosa, la revisión fue super facil!

👍

Clarissa0512 committed:
Arreglando href.

Aquí hago algunnos cambios con relación a un href de un link.

Clarissa0512 committed:
Arreglando el hr

Aquí arreglo el hr porque no hay que abrirlo y cerrarlo, sino que cierra en el mismo tag.

misscs commented:
He visto outra cosa 🙊 Debes cambiar `coehlo.html` a `index.html` como es la entrada para el servido

misscs commented on coelho.html:
```diff
@@ -30,10 +30,7 @@
-		  <li><a href="file:///C:/Myhtml-C/my-proyecto/yoko-kitchen/autobiografia.html">
-		  Clarissa Rodriguez Dionicio</a></li>
-	  </ul>
+	    <p>&copy; Clarissa R. D</p>
```
👍

misscs commented on coelho.html:
```diff
@@ -11,21 +11,21 @@
 		  <h1><i>Estos son escritos</i><em>de</em><b>Paulo Coelho</b></h1>
   </header>
-		  <hr></hr>
+		  <hr />
```
En html5, no tenemos que incluir el slash. `hr` es como `img` y `br`.  

Todas estas son válidos en html5

```
<hr>
<br>
<img src="" alt="">
```

Clarissa0512 committed:
Indentando los elementos.

Aquí indento los tags de manera organizada para su mejor lectura.

Clarissa0512 committed:
Arreglando orden de códigos.

Aquí indento una nueva vez los elementos.

Clarissa0512 committed:
Cambiar nombre.

Aquí cambio el nombre del archivo a index.

Clarissa0512 committed:
Eliminar archivo

Aquí elimino el archivo.

Clarissa0512 committed:
Arreglar el hr

Ahora arreglo el elemento hr.
